showcase/fastq-bio.md

```diff
@@ -2,18 +2,19 @@
 name: Fastq.bio
 logo_url: /assets/showcase-assets/fastq-bio.jpg
 website: http://www.fastq.bio/v2.0.1/index.html
-description: Figma is a collaborative interface design tool, powered by WebAssembly.
+description: Genomics analysis web tool that saw a 20X speedup thanks to WebAssembly
 keywords: c c++ genomics bio analytics data analysis math emscripten infomatics production
 ---
 
-fastq.bio, an interactive genomics web tool for analyzing genmoics data.
+fastq.bio, an interactive genomics web tool for analyzing genomics data.
 
 ![Fastq dot bio ScreenShot](/assets/showcase-assets/fastq-bio.jpg)
 
-fastq.bio uses WebAssembly, as the current implementation had a re-write of a C library in JavaScript so that it may run in the browser. However, this leads to many errors and a huge developement cost. With WebAssembly they were able to port the industry standard C Library to WebAssembly, to then be run in the browser. Not only did this lead to greater accuracy, however, it lead to a 10X speed up and better user experience for the analysis tool!
+fastq.bio was originally implemented as a JavaScript rewrite of a C library so that it could run in the browser. However, this approach had the potential for introducing errors and imposed a huge cost on development. With WebAssembly, they were able to port the industry standard C library so it could run in the browser. Not only did this reduce the risk for errors, but it also led to a 20X speed-up and a better user experience.
 
 The talk listed in the additional resources is highly reccomended!
 
 ## Additional Resources
 
-- [fastq.bio, an interactive genomics web tool, Robert Aboukhalil](https://youtu.be/lsqCrFCZOOw)
+- [fastq.bio, an interactive genomics web tool, Robert Aboukhalil](https://youtu.be/o-2PLhMVBYU)
```

Comment:
I updated the talk to point to my PerfMatters talk, just because it has a more entertaining intro :D